src/main/java/com/nike/cerberus/util/StackNameConverter.java

```diff
@@ -0,0 +1,12 @@
+package com.nike.cerberus.util;
+
+import com.beust.jcommander.IStringConverter;
+import com.nike.cerberus.domain.environment.Stack;
+
+public class StackNameConverter implements IStringConverter<Stack> {
```

Comment:
nit: should this be called `StackConverter`

Reply:
Done